docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -3996,6 +3996,65 @@ Next, you need to configure the Alarm Settings.  Under *Configure > Alarm Settin
 
 Finally, enable the WIPS feature on the controller.  Under *Configure > WIPS > Intrusion Detection and Prevention*, make sure both box are selected, click Apply. 
 
+Web authentication
+^^^^^^^^^^^^^^^^^^
+
+In order to use PacketFence as an external captive portal for web authentication, you will to configure first your radius authentication and accounting server (see steps above).
+
+[float]
+Hotspot configuration
++++++++++++++++++++++
+
+Then you will need to configure the Hotspot service profile to redirect devices to your PacketFence portal. Go on the ZoneDirector administration web page to the section *Configure->Hotspot Services->Create New*
```

Comment:
You can't really start a sentence with "Then"... especially when it is a new paragraph of a new section...